MonoGame.Framework/Graphics/States/BlendState.cs

```diff
@@ -241,7 +241,21 @@ static BlendState()
 	    internal BlendState Clone()
 	    {
 	        return new BlendState(this);
-	    }
-	}
+	    }
+
+        partial void PlatformDispose();
+
+        protected override void Dispose(bool disposing)
+        {
+            if (!IsDisposed)
+            {
+                for (int i = 0; i < _targetBlendState.Length; ++i)
+                    _targetBlendState[i] = null;
+
+                PlatformDispose();
+            }
+            base.Dispose(disposing);
+        }
+    }

```

Comment:
@KonajuGames - Could you apply this same `PlatformDispose` technique to all the other render states?  Just a little weird that the others just override `Dispose(bool)` now.  Ran into this on console.